src/services/HotWalletService.ts

```diff
@@ -48,10 +51,14 @@ class HotWalletService {
 
       const toBalance = await provider.getBalance(to)
       if (toBalance.lt(utils.parseEther(amount.toString()))) {
-        throw new Error(`Address ${to} does not have a high enough balance (${toBalance} to transfer ${amount} eth out`)
+        throw new Error(
+          `Address ${to} does not have a high enough balance (${toBalance} to transfer ${amount} eth out`
+        )
       }
 
-      throw new Error(`Transfers from the web3 wallet back to the exchange wallet are not supported yet.`)
+      throw new Error(
```

Comment:
This error is going to surface in our Testflight app, correct?

If so, we should probably also update that app to remove the logic that attempts to auto-transfer.

Reply:
I have a ticket to update the demo app to use Sepolia (https://linear.app/portal-hq/issue/ENG-1883/remove-goerli-usage-in-favor-of-sepolia). Is that sufficient or are you suggesting to remove the auto-transfer funds logic from the demo app since we haven't accrued sepolia test eth on the shared Portal wallet?